templates/csv_management/user_reservations.txt

```diff
@@ -0,0 +1,3 @@
+"Movie", "Playing time", "Cinema", "Hall", "Seat"
+{% for reservation in data %}"{{ reservation.details.movie|addslashes }}", "{{ reservation.details.playing_time|addslashes }}", "{{ reservation.details.cinema|addslashes }}", "{{ reservation.seat.hall|addslashes }}", "{{ reservation.seat.code|addslashes }}"
```

Comment:
Not a big deal, but you could remove the empty spaces between the data to avoid future issues.